### DISCUSSION TITLE
Eagleye 530s | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1207] [Q&A]
# Eagleye 530s | Seeed Studio Wiki

Eagleye_530s

https://wiki.seeedstudio.com/Eagleye_530s/

**Comment by souldiver** [comment]
If someone looking for a 530s Developer Kit (unused) -> short message at Github @souldiver https://github.com/souldiver or mekki1@gmx.net
